runtime/src/builtins.rs

```diff
@@ -51,63 +51,6 @@ macro_rules! with_program_logging {
     };
 }
 
-/// State transition enum used for adding and removing builtin programs through
```

Comment:
This block and the next are purely cosmetic, right?  Hard to follow major cosmetics in a functional change set